### PR TITLE
CONTRIBUTING GUIDELINES

### PR DESCRIPTION
**Pull request by anam140**
...

### CODE REVIEW

**Review by Mihir-Karbelkar — CHANGES_REQUESTED** [review]
Since this is a markdown  file you have the option of formatting it properly for example #CONTRIBUTING GUIDELINES would make it look like a heading and there are many more formatting options which you can use. You can follow this tutorial https://www.markdowntutorial.com/. There are some typos too which you can fix. Otherwise a great diff.

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Do you want to contribute to the Caffeineaholics/Login ??
+Join Caffeinoholics org .. 
+IT  is a volunteer effort. We encourage you to pitch in and join the team!
+
+Do you have questions?? 
+Ask any question  in discord. 
+Thanks! 
+
+CAFFEINEAHOLICS! Team!
```
Comment (Mihir-Karbelkar) [inline Contributing.md]: The Caffeineaholics Team

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+> If you're a project maintainer, you can assign the issue to someone, add it to a project board, associate it with a milestone,
+or apply a label.
+> When you're finished, click Submit new issue.
+
+Do you intend to add a new feature or change an existing one?
+Suggest your change in discussions or discord and   start writing code.
+
+Do you want to contribute to the Caffeineaholics/Login ??
+Join Caffeinoholics org .. 
+IT  is a volunteer effort. We encourage you to pitch in and join the team!
```
Comment (Mihir-Karbelkar) [inline Contributing.md]: It* typo

**Comment by Mihir-Karbelkar** [comment]
Also the file name should be all caps

**Review by Mihir-Karbelkar — CHANGES_REQUESTED** [review]
Please use markdown. There are places where you can use markdown like headings and bullets etc,
Also this pull request only requires one commit you will have to "Squash" your commits into one.

**Comment by Parimal7** [comment]
Kindly squash your commits into one, you can read more about it [here](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History). Read the section under squashing commits.

**Pull request closed without merging**